ast/dml_test.go

```diff
@@ -15,7 +15,9 @@ package ast_test
 
 import (
 	. "github.com/pingcap/check"
+	"github.com/pingcap/parser"
 	. "github.com/pingcap/parser/ast"
+	"strings"
```

Comment:
```suggestion
	"strings"
	
	. "github.com/pingcap/check"
	"github.com/pingcap/parser"
	. "github.com/pingcap/parser/ast"
```

We separate the interior imports and exterior imports with blank lines